p5js/NeuralNetwork/NeuralNetwork_titanic/sketch.js

```diff
@@ -1,45 +1,38 @@
-let classifier;
-let data;
-let normalizedData;
+let nn;
 let submitButton;
+let nnOptions = {
+  dataUrl: 'data/titanic_train-experimental.csv',
+  inputs: ['age', 'fare', 'is_female'],
+  outputs: ['survived_string'],
+  task: 'classification',
+  epochs: 50,
```

Comment:
I might leave `epochs` out of the example and just use a default. . or I think this might make more sense as an argument to the `train()` function since that's when you would be deciding total epochs?

Reply:
Cool! Yep, this is resolved.